packages/gravity/proto-guard/data/registry.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "name": "hypergraph",
+    "version": "1",
+    "dataRoot": "snapshots/1"
+  },
+  {
+    "name": "automerge",
+    "version": "2",
+    "dataRoot": "snapshots/2/snapshot",
+    "jsonDataPath": "snapshots/2/expected.json"
+  },
+  {
+    "name": "levelDB",
```

Comment:
I dont think leveldb is a good name - all snapshots after this will also technically be using leveldb. Perhaps name them after the semver versions or the month names